tests_integration/__tests__/__snapshots__/piped-output.js.snap

```diff
@@ -0,0 +1,15 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`no file diffs with --list-different + formatted file 1`] = `Array []`;
+
+exports[`no file diffs with --list-different + formatted file 2`] = `Array []`;
+
+exports[`output with --list-different + unformatted differs when piped 1`] = `
+"unformated.js[2K[1Gunformated.js
```

Comment:
I thought this was a color code. Oh well, `--no-color` doesn't hurt anyway. I guess it's from moving the cursor.

Reply:
Yes, and that's exactly the problem I had when piping the output: The next command in the pipe complained about that... :)